django/db/models/fields/__init__.py

```diff
@@ -950,7 +950,11 @@ def formfield(self, **kwargs):
         return None
 
 
-class BigAutoField(AutoField):
+# Aliased for backwards compatibility.
+AutoField = SmallAutoField
```

Comment:
are all operations in all pre-existing migrations definitely going to be fine with the class being renamed?

Reply:
@adamchainz should be, according to @andrewgodwin

Reply:
Pretty sure this will cause problems

https://github.com/django/django/blob/a375e911efcc78054d09ef31422e081507e56623/django/db/models/fields/__init__.py#L475